MAINTENANCE.md

```diff
@@ -79,42 +79,48 @@ There are 8 configuration files located in `.github/workflows`:
    In draft Pull Requests, only one job (Ubuntu + Python latest)
    is triggered to save on Continuous Integration resources.
 
+3. `ci_docs.yml` (Build documentation on Linux/macOS/Windows)
+
+   This is run on every commit to the *master* and Pull Request branches.
+   In draft Pull Request, only the job on Ubuntu is triggerd to save on
+   Continuous Integration resources.
```

Comment:
We say Linux/macOS/Windows and then use Ubuntu. Maybe just use Linux in case people don't know Ubuntu is a Linux distribution.

```suggestion
   In draft Pull Requests, only the job on Linux is triggered to save on
   Continuous Integration resources.
```